consensus/src/runway/mod.rs

```diff
@@ -171,18 +172,67 @@ impl<'a, H: Hasher> RunwayStatus<'a, H> {
     }
 }
 
+fn short_report(rounds_behind: Round, missing_coords: usize) -> String {
```

Comment:
nit: I somehow dont like that this is not a priv funcion of TunwayStatus struct

Reply:
done &